curl.ml

```diff
@@ -641,6 +641,26 @@ external get_headers_rev: t -> headerOrigin list -> int -> (string * string) lis
 
 let get_headers t origins ~request = get_headers_rev t origins request |> List.rev
 
+(** WebSocket support *)
+type curlWSFlag =
+  | CURLWS_TEXT
+  | CURLWS_BINARY
+  | CURLWS_CONT
+  | CURLWS_CLOSE
+  | CURLWS_PING
+  | CURLWS_PONG
+  | CURLWS_OFFSET
+
+type curlWSFrame = {
+  age: int;
+  flags: curlWSFlag list;
+  offset: int64;
+  bytesleft: int64;
```

Comment:
please convert to int
32-bit arch is not a problem - the string size limit will not allow to handle big frames anyway there